docs/src/Guides/99 Migration From D.py.md

```diff
@@ -1,4 +1,25 @@
 # Migration From discord.py
 
-!!! danger "Notice"
-    This page is a placeholder, and will be populated at a later date.
+1. Dis-Snek requires python 3.10 (as compared to dpy's 3.5), you may need to upgrade python.
+  - If you see `ERROR: Could not find a version that satisfies the requirement dis_snek (from versions: none)` when trying to `pip install dis-snek`, this is your problem.
+2. Classes/Models
+  - Your client is `dis_snek.Snake`.  (Note that commands are a first-class feature, so this is a replacement to both `discord.Client` and `discord.ext.commands.Bot`)
+  - Cogs are `Scales`.
+  - `Member` is not a subclass of `User`, if you're using `isinstance`, you'll want to check both explicitly.
+3. Scales (Cogs)
+  - These work mostly the same, with a few notable changes:
+  - Your setup function doesn't need to do `bot.add_cog()`.  Simply call `MyCog(bot)`, and it'll automatically register itself.
+  - Scales already define `self.bot`, you don't need to do that in your `__init__` function.
+  - For a full example, see [here](/Guides/20 Scales/)
+4. Event handlers
+  - Register event handlers with `@listen`
```

Comment:
Imo `@dis_snek.listen` instead

Reply:
I went back and forth on this one. Anyone else have opinions?

Reply:
I think `@dis_snek.listen` is a bit clearer since otherwise it seems rather vague.

Reply:
D.py users may think that its `@Snake.listen`

Reply:
Yeah, good catch